Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcherService.h

```diff
@@ -136,9 +136,12 @@ extern NSString *const kGTMSessionFetcherServiceSessionKey;
 // Common method for fetcher creation.
 //
 // -fetcherWithRequest:fetcherClass: may be overridden to customize creation of
-// fetchers.  This is the ONLY method in the GTMSessionFetcher library intended to
+// fetchers.  These are the ONLY method in the GTMSessionFetcher library intended to
 // be overridden.
 - (id)fetcherWithRequest:(NSURLRequest *)request fetcherClass:(Class)fetcherClass;
+- (id)fetcherWithRequest:(NSURLRequest *)request
+              fetcherClass:(Class)fetcherClass
+    stopTriggersCompletion:(BOOL)stopTriggersCompletion;
```

Comment:
I was thinking we just make this a property on the service and then you don't need a new api, it just gets copied out like all the other properties that get carried over?

Reply:
That doesn't work if you want to use the service to call both sets of APIs. I suppose I could change Firebase to add a second service just for usage from async APIs?

Reply:
> Done.

Just checking, did that mean you had another commit to also push?  I don't think one appeared after this comment.

Reply:
Yep. I didn't notice that I typo'd the `git push` command. Thanks.